### ISSUE TITLE
Dependency problem while installing 1.0.214

### ISSUE THREAD

**Issue by robertbiskup** [IFCjs/web-ifc-viewer#207]
Hi, while installing web-ifc-viewer by npm there are problems with dependencies (web-ifc-viewer/web-ifc-three) "three-mesh-bvh": "^0.5.11"/"0.5.21", "three": "^0.135.0"/"^0.149.0"

**Comment by bricekouetcheu** [comment]
hello. did you find a solution? I am facing the same issue

**Comment by RASRC** [comment]
I had this issue too. I thin'k it's in progress there is another post talking about the problem #188

**Comment by Nomeon** [comment]
The dependency issue is caused by the change in viewer/package.json:

\- "web-ifc": "^0.0.35",
\- "web-ifc-three": "^0.0.118"
\+ "web-ifc": "^0.0.39",
\+ "web-ifc-three": "^0.0.124"

Web-ifc-three expects 'three' version ^0.149.0, whereas web-ifc-viewer is still on version ^0.135.0. Same goes for three-mesh-bvh. This was the 'breaking' change from web-ifc-viewer 1.0.213 to 1.0.214.

I'm not sure, but if web-ifc-three is working fine on 0.149.0, isn't it as simple as bumping the version in web-ifc-viewer to 0.149.0 too @agviegas?

**Comment by agviegas** [comment]
@Nomeon unfortunately, that's not the case. Updating web-ifc-viewer is complex, and because it will be substituted by [components](https://github.com/ifcjs/components) very soon, we are considering if spending the time updating web-ifc-viewer three.js dependency is worth. 🤔

**Comment by Nomeon** [comment]
@agviegas Understandable, especially if you will change the architecture to a component-based one, I wouldn't spend the time on it either.

**Comment by agviegas** [comment]
Yep! Very soon this summer we'll publish a brand new documentation with many, many tutorials, the whole new API, etc. So hopefully everything will be much easier to follow and use 🙂
